lldb/test/API/macosx/tbi-honored/TestTBIHonored.py

```diff
@@ -0,0 +1,49 @@
+"""Test that lldb on Darwin ignores metadata in the top byte of addresses."""
+
+import os
+import re
+import subprocess
+
+import lldb
+from lldbsuite.test.decorators import *
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test import lldbutil
+
+
+class TestTBIHonored(TestBase):
+    @no_debug_info_test
+    @skipUnlessDarwin
+    @skipIf(archs=no_match(["arm64", "arm64e"]))
+    @skipIfRemote
+    def do_variable_access_tests(self, frame):
```

Comment:
Just in case we mess up really badly, it is worth checking that pb.bytes[7] is in fact 0xfe first.

That would also functions as documentation implying "despite this being 0xfe we can... <rest of expects>".